src/core/remote.js

```diff
@@ -1610,7 +1610,7 @@ Remote.prototype.requestBookOffers = function(options_, callback_) {
   const request = new Request(this, 'book_offers');
 
   request.message.taker_gets = {
-    currency: Currency.json_rewrite(gets.currency, {force_hex: true})
```

Comment:
Not sure. On droid. Try doing
Currency.from_json(interestBearingCur).to_json() should say a lot.